server/routes/email-router.js

```diff
@@ -0,0 +1,16 @@
+require('dotenv/config')
+const { Router } = require('express')
+const sgMail = require('@sendgrid/mail')
+
+module.exports = function emailRouter() {
+
+  const router = new Router()
+
+  router.post('/', (req, res, next) => {
+    sgMail.setApiKey(process.env.SENDGRID_API_KEY)
+    const msg = req.body
+    sgMail.send(msg)
+      .catch(err => next(err))
```

Comment:
You never respond in this handler, so even if the mail gets sent, the client doesn't know and just waits forever until it times out. Try testing this with HTTPie.